code/modules/hydroponics/spreading/spreading_growth.dm

```diff
@@ -19,9 +19,13 @@
 
 		//We check zdest, not floor, for existing plants
 		if((locate(/obj/effect/plant) in zdest.contents) || (locate(/obj/effect/dead_plant) in zdest.contents) )
-
 			continue
 
+		//If our floor is open space, so let's check turf below
+		if(istype(floor, /turf/simulated/open))
```

Comment:
this is what the zdest thing is for, it gets the turf below in the case of openspace tiles.

it may be buggy, if so fix it, but don't make a redundant bit of code like this